### DISCUSSION TITLE
Verify validity of generated import-profile and catalog

### DISCUSSION THREAD

**Discussion by telosBA** [usnistgov/OSCAL#1326] [Q&A]
Our team has generated an import-profile based on the controls that will be used in our SSP. Attached you will find the import-profile as well as the associated catalog in .txt format. We would like to verify that the methodology we are using is acceptable and that the files are in an acceptable format.

We would also like to know if the expected submission is a single .json with all of these objects included, or separate files for each object
[OscalProfile.txt](https://github.com/usnistgov/OSCAL/files/9002285/OscalProfile.txt)
[oscalCatalog.txt](https://github.com/usnistgov/OSCAL/files/9002286/oscalCatalog.txt)
.

**Comment by david-waltermire** [comment]
@telosBA There are issues with the files you provided. Can we have a phone call to discuss this? PM me on Gitter to coordinate.

**Reply by telosBA** [reply]
Sent
